### PR TITLE
Feature/json export

### PR DESCRIPTION
**Pull request by 902seanryan**
Marking as draft for now, as it builds on https://github.com/SpringRoll/SpringRollStudio/pull/31

Once that is merged I'll move this to a real PR.

### CODE REVIEW

File: src/renderer/class/Directory.js [hunk 1]
```diff
@@ -106,6 +109,8 @@ export default class Directory {
   selectByFile(file) {
     const index = this.getFileIndex(file);
 
+    //const index = this.files.indexOf(file);
```
Comment (aberkie) [inline src/renderer/class/Directory.js:112]: Do we need this?

File: src/renderer/components/FileExplorer.vue [hunk 2]
```diff
@@ -1,140 +1,176 @@
...
-    </v-dialog>
-    <!-- <v-dialog v-else v-model="dialog" width="500">
-      <template v-slot:activator="{ on }">
-        <v-btn
-          color="accent"
-          class="v-btn explorer__input --file font-semi-bold font-16"
-          v-on="{ on }"
-        >
-          Import Files
+<div class="explorer">
```
Comment (aberkie) [inline src/renderer/components/FileExplorer.vue:2]: Looks like there might be linting issues from GitHub actions in this file

File: src/renderer/components/JsonPreview.vue [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
-    <v-jsoneditor ref="jsonEditor" class="json__editor" :options="options" :plus="false" height="400px" />
+    <v-jsoneditor ref="jsonEditor" class="json__editor" :options="options" :plus="false" height="446px" />
     <div class="json__button-group">
+      <v-dialog v-model="saveErrorDialog" width="500">
+        <v-card>
+          <v-card-title class="error" primary-title>
+            <h2 class="font-semi-bold json__dialog-title">Warning</h2>
+          </v-card-title>
+          <v-card-text>
+            <span class="font-16">There are errors in the caption JSON. It is recommended you correct those before saving, otherwise your changes to those captions will not be saved</span>
```
Comment (aberkie) [inline src/renderer/components/JsonPreview.vue:11]: `captions will not be saved.`

**Review by aberkie — CHANGES_REQUESTED** [review]

File: src/renderer/class/Directory.js (hunk 1 above)
Comment (902seanryan) [reply src/renderer/class/Directory.js:112]: We don't! And I promised Corinne I'd have it removed, thanks for catching it 👍

**Review by deycorinne — APPROVED** [review]

**Review by aberkie — APPROVED** [review]

**Review by BaileyBrightman — APPROVED** [review]

**Pull request merged**